frontend/copr-frontend.spec

```diff
@@ -138,23 +138,28 @@ Recommends: fedora-messaging
 Requires: js-jquery
 Requires: python3-anytree
 Requires: python3-click
-Requires: python3-CommonMark
 Requires: python3-alembic
 Requires: python3-blinker
 Requires: python3-copr-common >= %copr_common_version
 Requires: python3-dateutil
 Requires: python3-email-validator
 Requires: python3-flask
-%if 0%{?fedora} >= 31
-Requires: python3-flask-caching
+%if 0%{?fedora} >= 31 || 0%{?openEuler}
+Requires: python3dist(flask-caching)
 %else
 Requires: python3-flask-cache
 %endif
-Requires: python3-flask-openid
-Requires: python3-flask-sqlalchemy
+Requires: python3dist(commonmark)
+Requires: python3dist(flask-openid)
+Requires: python3dist(flask-sqlalchemy)
+Requires: python3dist(sqlalchemy-utils)
+Requires: python3dist(pylibravatar)
+Requires: python3dist(wtforms) >= 2.2.1
+Requires: python3dist(flask-wtf)
+Requires: python3dist(xstatic-datatables)
+Requires: python3dist(xstatic-patternfly)
+Requires: python3dist(xstatic-bootstrap-scss)
```

Comment:
This is hard to detect at build time;  but ` xstatic-bootstrap-scss-common` do not provide `python3dist`, are you sure this works on openEuler?